src/Agent.Plugins/Telemetry/TelemetryConstants.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Agent.Plugins.Log.TestResultParser.Plugin
+{
+    public class TelemetryConstants
+    {
+        public const string PluginInitialized = "PluginInitialized";
```

Comment:
>public const string [](start = 8, length = 19)

will this be accessed from different dll? if yes lets chat

Reply:
same dll.